localstack-core/localstack/services/events/models.py

```diff
@@ -237,4 +237,4 @@ class EventsStore(BaseStore):
     TAGS: TaggingService = CrossRegionAttribute(default=TaggingService)
 
 
-events_store = AccountRegionBundle("events", EventsStore)
+events_stores = AccountRegionBundle("events", EventsStore)
```

Comment:
comment: this was the main reason why the store wasn't being picked up by the state visitor as the `_stores` suffix is a requirement for automatic detection.